inst/python_cmd.m

```diff
@@ -138,11 +138,8 @@
   end
 
   if (strcmp(A{1}, 'COMMAND_ERROR_PYTHON'))
-    errlineno = A{3} - LinesBeforeCmdBlock - 1;
-    errstr = sprintf(['Python raised exception, maybe at line %d of code ' ...
-                      'block (assuming popen2)\n' ...
-                      '%s'], errlineno, A{2});
-    error('OctSymPy:python_cmd:exception', errstr);
+    errlineno = A{3} - db.prelines - LinesBeforeCmdBlock;
+    error(strcat(A{2}, ' (line: ', mat2str(errlineno), ')'));
```

Comment:
I liked that it specified it was a python error.  How about keep it like it was before, something like:
````
error: Python exception near line 1 of the code block\n
NameError: hello
````
(File a separate bug if you feel strongly about fitting it on one line).